spec/facility_spec.rb

```diff
@@ -1,26 +1,156 @@
+    @facility_1 = Facility.new({name: 'DMV Tremont Branch', address: '2855 Tremont Place Suite 118 Denver CO 80205', phone: '(720) 865-4600'})
+    @facility_2 = Facility.new({name: 'DMV Northeast Branch', address: '4685 Peoria Street Suite 101 Denver CO 80239', phone: '(720) 865-4600'})
+    @cruz = Vehicle.new({vin: '123456789abcdefgh', year: 2012, make: 'Chevrolet', model: 'Cruz', engine: :ice} )
+    @bolt = Vehicle.new({vin: '987654321abcdefgh', year: 2019, make: 'Chevrolet', model: 'Bolt', engine: :ev} )
+    @camaro = Vehicle.new({vin: '1a2b3c4d5e6f', year: 1969, make: 'Chevrolet', model: 'Camaro', engine: :ice} )
+    @registrant_1 = Registrant.new('Bruce', 18, true)  
+    @registrant_2 = Registrant.new('Penny', 16)
+    @registrant_3 = Registrant.new('Tucker', 15)
   end
+   
   describe '#initialize' do
     it 'can initialize' do
-      expect(@facility).to be_an_instance_of(Facility)
+      expect(@facility).to be_an_instance_of(Facility)  
       expect(@facility.name).to eq('DMV Tremont Branch')
       expect(@facility.address).to eq('2855 Tremont Place Suite 118 Denver CO 80205')
       expect(@facility.phone).to eq('(720) 865-4600')
       expect(@facility.services).to eq([])
+      expect(@facility_1).to be_an_instance_of(Facility)  
+      expect(@facility_1.name).to eq('DMV Tremont Branch')
+      expect(@facility_1.address).to eq('2855 Tremont Place Suite 118 Denver CO 80205')
+      expect(@facility_1.phone).to eq('(720) 865-4600')
+      expect(@facility_1.services).to eq([])
+      expect(@facility_2).to be_an_instance_of(Facility)  
+      expect(@facility_2.name).to eq('DMV Northeast Branch')
+      expect(@facility_2.address).to eq('4685 Peoria Street Suite 101 Denver CO 80239')
+      expect(@facility_2.phone).to eq('(720) 865-4600')
+      expect(@facility_2.services).to eq([])
     end
   end
 
-  describe '#add service' do
+  describe '#add_service' do
     it 'can add available services' do
       expect(@facility.services).to eq([])
       @facility.add_service('New Drivers License')
       @facility.add_service('Renew Drivers License')
-      @facility.add_service('Vehicle Registration')
-      expect(@facility.services).to eq(['New Drivers License', 'Renew Drivers License', 'Vehicle Registration'])
+      @facility_1.add_service('New Drivers License')
+      @facility_1.add_service('Vehicle Registration')
+      @facility_2.add_service('New Drivers License')
+      @facility_2.add_service('Renew Drivers License')
+      @facility_2.add_service('Vehicle Registration')
+      expect(@facility.services).to eq(['New Drivers License', 'Renew Drivers License'])
+      expect(@facility_1.services).to eq(['New Drivers License', 'Vehicle Registration'])
+      expect(@facility_2.services).to eq(['New Drivers License', 'Renew Drivers License', 'Vehicle Registration'])
+    end   
+  end
+
+  describe '#register_vehicle' do
+    it 'can register vehicles' do
```

Comment:
I would break this up into different scenarios just so it's easier to read and troubleshoot:
- it 'can register antique vehicles'
- it 'can register electric vehiecles'
- it 'can register basic vehicles as long as service is available'

You're covering all the scenarios but in one test case it just gets a bit long